src/woff2/collection.rs

```diff
@@ -3,12 +3,14 @@ use crate::error::ParseError;
 use crate::woff2::{PackedU16, TableDirectoryEntry, Woff2Font};
 
 #[derive(Debug)]
+#[allow(unused)]
 pub struct Directory {
     version: u32,
     entries: Vec<FontEntry>,
 }
 
 #[derive(Debug)]
+#[allow(unused)]
```

Comment:
Can these go on the unused fields themselves?

Reply:
I'm used to doing attributes only at the top level. Done.